apps/web/src/components/team-builder/v2/builder.module.css

```
@@ -1056,35 +1101,68 @@
+:global(.mvline-calc-inline--ko2) { color: var(--ko-amber2-fg); }
+:global(.mvline-calc-inline--ko3) { color: var(--ko-yellow-fg); }
+:global(.mvline-calc-inline--ko4) { color: var(--muted-foreground); }
+
+:global(.mvline-calc-inline__tier) {
+  font-size: 8.5px;
+  font-weight: 800;
+  letter-spacing: 0.05em;
+  text-transform: uppercase;
+}
+
+:global(.mvline-calc-inline__range) {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)